### PR TITLE
Added App Icon

### PR DESCRIPTION
**Pull request by TinkerTavern**
Simple change to make the app icon the Turing Roulette icon instead of the default one

### CODE REVIEW

**Review by emberdex — CHANGES_REQUESTED** [review]
Don't use PNG icons as app icons. This breaks adaptive icons in newer versions of Android and results in an icon with borders :(

Instead, you should create an image asset (right click on the `drawable` folder in `res` and create a new Image Asset), and follow the wizard there, using the existing drawable `ic_turing_icon_fg` as the foreground and the purple colour in `colors.xml` as the background.

Make sure to revert your old changes too :)

**Comment by TinkerTavern** [comment]
> Don't use PNG icons as app icons. This breaks adaptive icons in newer versions of Android and results in an icon with borders :(

Oops! Didn't realise that was what it was doing. It should be done the right way now

> Make sure to revert your old changes too :)

Done!

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
```
Comment (emberdex) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: You might be able to change this to `@color/colorPrimary` and remove the `ic_launcher_background.xml` file as a result. Can you give that a try and let me know if it works please? 😄

Comment (TinkerTavern) [reply app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: Yup! Just switched it over now

**Review by emberdex — APPROVED** [review]
Thanks for doing that :)

**Pull request merged**